opm/parser/eclipse/EclipseState/Schedule/Well.cpp

```diff
@@ -303,11 +526,109 @@ namespace Opm {
         }
         return wellNameInPattern;
     }
+    bool Well::isMultiSegment() const {
+        return m_is_multi_segment;
+    }
+
+    void Well::setMultiSegment(const bool is_multi_segment) {
+        m_is_multi_segment = is_multi_segment;
```

Comment:
Can we infer the multisegment status of the well by just looking at the size of the segmenteset:
```C++
bool Well::isMultiSegemnt() const {
    // Do not think the DynamicState->getLast() exists - you can add it.
    if (m_segmentset->getLast()->size() > 0)
       return true;
    else
      return false;
}
```